tools/distro-json-generate.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs-extra';
+import got from 'got';
+import shell from 'shelljs';
+
+const url = 'https://debian.pages.debian.net/distro-info-data/ubuntu.csv';
+
+/**
+ * @param {string} raw
+ * @returns {string}
+ */
```

Comment:
not needed for TS. if used please extend to full description, so it's clear what the function is for.